Release-Notes-R6.md

```diff
@@ -1,8 +1,134 @@
+with [OpenTofu](https://opentofu.org/),
+an open-source, community-driven IaC tool. Users don't have to worry anymore.
+
+### ClusterClass
+
+From [#600](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/600), this repository uses CAPI
+[ClusterClass](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-class/) feature for the creation of
+clusters, see k8s [blog](https://kubernetes.io/blog/2021/10/08/capi-clusterclass-and-managed-topologies/) for overview.
+This feature is also used in the SCS [Cluster Stacks](https://github.com/SovereignCloudStack/cluster-stacks) - KaaS
+reference implementation v2.
+
+### Proxy
+
+[#418](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/418) is about support for corporate proxy
+in the Kubernetes clusters. In R6, users are able to specify e.g. `http_proxy = "http://10.10.10.10:3128"`
+and this proxy setting will be propagated to the management host as well as on the worker and control plane nodes,
+see [#620](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/620)
+and [#645](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/645) for details.
+There is also a `no_proxy` setting for configuring exceptions,
+[#651](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/651).
+
+### OVN LB
+
+From R4, LoadBalancer in front of ingress-nginx can be configured to use the OVN provider
+instead of the default Amphora provider. But, it was only a tech preview feature and not recommended.
+In R6 [#687](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/687),
+after all upstream issues were resolved, we removed the tech preview flag and carefully tested it with success.
+OVN LoadBalancer can be enabled by setting `use_ovn_lb_provider = "true"` or `use_ovn_lb_provider = "auto"`.
+In the future [#706](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/706), it will be
+enabled by default.
+
+### Renovate
+
+Automated dependency updates and [renovate bot](https://docs.renovatebot.com/)
+are part of this repository from [#596](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/596)
+where config for CAPI and CAPO was added.
+Later it was extended for calico in [#622](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/622),
+terraform-provider-openstack in [#633](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/633)
+and for k9s in [#629](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/629).
+More will come in the future, see [#577](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/577).
+
+### Restrict access to the management server
+
+From [#599](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/599),
+ssh port 22 of the management server can be restricted with a list of cidrs.
+By default, there is no restriction `restrict_mgmt_server = ["0.0.0.0/0"]`
```

Comment:
consider: 

From [#599](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/599), it's now possible to restrict port 22 SSH access to the management server using a whitelist of CIDRs. By default, there are no restrictions, as indicated by restrict_mgmt_server = ["0.0.0.0/0"].

Reply:
Thank you, please see 90ca44f5cfa538c7aa292f4c2069e793c140412e